tests/test_actors.py

```diff
@@ -471,3 +553,21 @@ def test_cli_show_family_when_get_by_name_makes_expected_call(
     )
     httpserver_auth.check()
     assert result.exit_code == 0
+
+
+def test_cli_actor_update_makes_expected_call(
```

Comment:
Would be nice to test all possible args, maybe one test that clears them all and one that sets them all?

Reply:
Added tests.